### PR TITLE
Update jetty.jsonnet

### PR DESCRIPTION
**Pull request by jmcc0nn3ll**
rename the jetty.test.policy repo to jetty-test-policy to test some things

### CODE REVIEW

**Review by netomi — APPROVED** [review]

File: otterdog/jetty.jsonnet [hunk 1]
```diff
@@ -310,7 +310,8 @@ orgs.newOrg('jetty') {
         actions_can_approve_pull_request_reviews: false,
       },
     },
-    orgs.newRepo('jetty.test.policy') {
+    orgs.newRepo('jetty-test-policy') {
+      aliases: ["jetty.test.policy"],
```
Comment (joakime) [inline otterdog/jetty.jsonnet:314]: Is this entry something we have to keep permanently in the jetty.jsonnet now?

Comment (jmcc0nn3ll) [reply otterdog/jetty.jsonnet:314]: I am wondering that as well, for example if we moved i-jetty over from jetty-project and then decided to rename it in the jetty org, would that disrupt follows and things. Or is the alias something that protects that?

**Comment by jmcc0nn3ll** [comment]
When does this take action? The repo was archived and I was wondering if that was an issue'

**Comment by netomi** [comment]
indeed renaming an archived repo is not possible, I did unarchive it and then apply, changes are now live.

**Pull request merged**